test/webrtc/ZombieRTCPeerConnection_spec.js

```diff
@@ -536,12 +536,43 @@ describe("A ZombieRTCPeerConnection", function () {
 		});
 	});
 
+	describe("removing a stream", function () {
+		describe("with an onremovestream handler", function () {
+			var onremovestream = Sinon.spy();
+
+			before(function (done) {
```

Comment:
Do you plan on adding other tests? Otherwise not sure why these are in the before block and not just in the test body.